src/orchestrator-tracing/src/lib.rs

```diff
@@ -354,6 +367,8 @@ impl NamespacedOrchestrator for NamespacedTracingOrchestrator {
                             .expect("opentelemetry-header had non-ascii value"),
                     ));
                 }
+                // TODO(guswynn): If `--sentry-dsn` is passed, but not `--opentelemetry-endpoint`,
+                // these are skipped for clusters.
```

Comment:
This seems like an easy fix—just pop this out of the `if let Some(endpoint) = opentelemetry_endpoint {` block?